DataLayer/SongLibrary/DownloadSongs.sh

```diff
@@ -3,3 +3,11 @@
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-06+Animal.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-11+Armageddon+It.mp3
 wget https://s3-us-west-2.amazonaws.com/ucsolarteam.onboardmedia.songs/1-14+Hysteria.mp3
+sudo apt-get install eyeD3
+eyeD3 --write-images=$PWD 1-14+Hysteria.mp3 1-06+Animal.mp3 1-14+Hysteria.mp3 01-Take-On-Me.mp3
```

Comment:
As I mentioned before, if we have a bunch of songs downloaded this command will get really really big. What I would recommend is to figure out how to get the name of the song, then run a bash for loop to grab of all the songs. Try consulting the `man` page of `eyeD3` for any info on getting the song name.

Reply:
Because the songs get downloaded to the directory immediately after each line gets executed, it might be easier to scan the directory for all files that end with .mp3 after they have all been downloaded and then get their album arts.

Reply:
Try the wildcard operator `*`

Reply:
The section for renaming songs will also get very large. Will need to figure out a way to rename them appropriately.